### ISSUE TITLE
bug(cast): resolve-name and lookup-address ENS commands do not work

### ISSUE THREAD

**Issue by emo-eth** [foundry-rs/foundry#7759]
### Component

Cast

### Have you ensured that all of these are up to date?

- [X] Foundry
- [X] Foundryup

### What version of Foundry are you on?

forge 0.2.0 (008922d 2024-04-23T00:22:15.514185000Z)

### What command(s) is the bug in?

cast lookup-address, cast

### Operating System

macOS (Apple Silicon)

### Describe the bug

These commands used to work. They no longer do.

```
❯ cast lookup-address 0x28679A1a632125fbBf7A68d850E50623194A709E
<null>
❯ cast lookup-address 0x333601a803CAc32B7D17A38d32c9728A93b422f4
<null>
❯ cast resolve-name emo
0x0000000000000000000000000000000000000000
❯ cast resolve-name slokh
0x0000000000000000000000000000000000000000
```

See: https://app.ens.domains/slokh.eth https://app.ens.domains/emo.eth

For whatever it's worth, when passing the --verify flag, which cast says does the following: "Perform a reverse lookup to verify that the name is correct."

```
❯ cast resolve-name -v emo
The application panicked (crashed).
Message:  assertion `left == right` failed: forward lookup verification failed. got , expected emo
  left: ""
 right: "emo"
```

**Comment by DaniPopes** [comment]
Note that just `emo` still won't work after #7765, we always required the full name string.

The error message would generically say "resolver not found", so this could be improved, or alternatively we can default any name that does not contain dots "." to ".eth".
I'll leave this for another PR.
